src/browser/search.py

```diff
@@ -1,25 +1,25 @@
-import requests
-from src.config import Config
+from enum import Enum
+# from src.config import Config
+from .search_engines import BingSearch , DuckDuckGoSearch
 
-class BingSearch:
+
+class SearchEngine(Enum):
+    BING = BingSearch()
+    DUCKDUCKGO = DuckDuckGoSearch()
+
+
+class WebSearch:
     def __init__(self):
-        self.config = Config()
-        self.bing_api_key = self.config.get_bing_api_key()
-        self.bing_api_endpoint = self.config.get_bing_api_endpoint()
+        # self.config = Config()
+        self.search_engine = SearchEngine.DUCKDUCKGO.value
```

Comment:
Couldn't We make to enable user to set search engine class via config ?
```suggestion
        self.search_engine = load_class(self.config.get_search_engine_class()) // replace load class method
```